enable/tests/tools/pyface/test_move_command_tool.py

```diff
@@ -47,13 +47,13 @@ def test_drag_component(self):
 
         # start moving the mouse
         mouse_move_event = self.mouse_move(
-            self.component, 145, 145, window=window
+            self.component, 145, 145, window=window, left_down=True
```

Comment:
These and others can be reverted now, right?

Reply:
they can but I think it is best to be explicit.

Some day we will want to remove dependence on `Undefined` being `True`.  I figure this is just a part of a longer term migration

Reply:
Ah right. Good point.